tools/PI/DevHome.PI/Models/AppRuntimeInfo.cs

```diff
@@ -2,84 +2,157 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Collections.ObjectModel;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Management;
+using System.Text.RegularExpressions;
 using CommunityToolkit.Mvvm.ComponentModel;
+using DevHome.PI.Helpers;
 using Microsoft.UI.Xaml;
+using Serilog;
 
 namespace DevHome.PI.Models;
 
 public partial class AppRuntimeInfo : ObservableObject
 {
+    private static readonly ILogger _log = Log.ForContext("SourceContext", nameof(AppRuntimeInfo));
+
+    private readonly string _unknownText = CommonHelper.GetLocalizedString("UnknownText");
+
+    private const string UwpCommandLinePattern = @"^(.*) -ServerName:(.*?)\.AppX(.*?)\.mca$";
```

Comment:
Can you provide a few examples of the command lines that will match this pattern?

Reply:
Added as a comment, also here:
        "C:\Program Files\WindowsApps\Microsoft.WindowsAlarms_11.2406.47.0_x64__8wekyb3d8bbwe\Time.exe" -ServerName:App.AppXq8avk61zazpy808ab5ppkf6taqp47km6.mca
        "C:\Program Files\WindowsApps\35455AndrewWhitechapel.uTaskManager_2309.21.1.0_x64__6rjrek5qak82t\uTaskManager.exe" -ServerName:uTaskManager.AppXjaq7n2ahxkbe1kpkhkxqhr5d0s2yr0pb.mca
        "C:\Foo\TestApps\UwpAea\UwpAea\bin\x86\Debug\AppX\UwpAea.exe" -ServerName:Blueberry.Pie.AppXnzm9t7zr5rgagha6146e9rgzyahj42xx.mca